bindings/pyroot/src/Pythonize.cxx

```diff
@@ -2478,7 +2478,7 @@ Bool_t PyROOT::Pythonize( PyObject* pyclass, const std::string& name )
 
    }
 
-   else if ( IsTemplatedSTLClass( name, "vector" ) ) {
+   else if ( IsTemplatedSTLClass( name, "vector" ) || IsTemplatedSTLClass( name, "ROOT::VecOps::RVec" )) {
```

Comment:
How could this be working on Linux if we were missing the whole pythonization for `RVec`?

Reply:
By chance? I can now verify that with this pythonisation everything works on osx but asArray is unusable on linux. Very nice headache.